docs/design/fabric-v2+/fpc-management.md

```diff
@@ -37,7 +37,7 @@ This command has the same requirements *when it is used for FPC chaincodes* as d
 
 ### Create Chaincode Enclave
 
-```peer lifecycle chaincode createenclave -n <chaincode id>```
+```peer lifecycle chaincode initEnclave -n <chaincode id>```
```

Comment:
all fabric commands are currently all lower-case, so this seems a bit odd. But i think it is consistent across the board ...
BTW: as mentioned elsewhere, we should also document the need for the peer param and in particular the new sgx parameter